setup.py

```diff
@@ -32,40 +32,36 @@ def about_package(init_posixpath: pathlib.Path) -> dict:
 
 # Extra dependencies for tools
 EXTRA_DEPENDENCIES = {
-                      'test': ['pytest>=5.2',
-                               'pytest-cov>=2.6',
-                               'hypothesis>=4.36.2',
-                               'attrs>=19.2.0'],
-                      'doc': ['sphinx',
-                              'travis-sphinx',
-                              'sphinx_rtd_theme']
-                      }
+    "test": ["pytest>=5.2", "pytest-cov>=2.6", "hypothesis>=4.36.2", "attrs>=19.2.0"],
+    "doc": ["sphinx", "travis-sphinx", "sphinx_rtd_theme"],
+}
 
 # The text of the README file
 with README.open("r") as docs:
     long_description = docs.read()
 
-# This call to setup() does all the work
 setup(
-    name=ABOUT_GENERIC_PARSER['__title__'],
-    version=ABOUT_GENERIC_PARSER['__version__'],
-    description=ABOUT_GENERIC_PARSER['__description__'],
+    name=ABOUT_GENERIC_PARSER["__title__"],
+    version=ABOUT_GENERIC_PARSER["__version__"],
+    description=ABOUT_GENERIC_PARSER["__description__"],
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url=ABOUT_GENERIC_PARSER['__url__'],
-    author=ABOUT_GENERIC_PARSER['__author__'],
-    author_email=ABOUT_GENERIC_PARSER['__author_email__'],
-    license=ABOUT_GENERIC_PARSER['__license__'],
+    url=ABOUT_GENERIC_PARSER["__url__"],
+    author=ABOUT_GENERIC_PARSER["__author__"],
+    author_email=ABOUT_GENERIC_PARSER["__author_email__"],
+    license=ABOUT_GENERIC_PARSER["__license__"],
     python_requires=">=3.6",
     classifiers=[
         "License :: OSI Approved :: MIT License",
-        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3 :: Only",
         "Programming Language :: Python :: 3.6",
```

Comment:
very nitpicky, but this should be gone now, no?

Reply:
It should indeed, good catch!